x-pack/plugins/searchprofiler/public/directives/_directives.scss

```diff
@@ -0,0 +1,148 @@
+.profilerDevTool {
+  
+  .panel {
+    background-color: inherit;
+    margin: 0;
+    padding: 0;
+    border: none;
+    border-bottom: 1px solid $euiColorDarkShade;
+  }
+
+  .shard-body {
+    padding-top: $euiSizeXS;
+    margin: 0;
+    margin-left: $euiSizeXS;
+  }
+
+  .panel-body-underline {
+    padding: $euiSize $euiSizeXS $euiSizeXS;
+    margin-top: $euiSize;
+    margin-left: $euiSizeS;
+  }
+
+  // Profile details treeview
+  .shard-details {
+    float: left;
+    margin-bottom: $euiSizeXS;
+
+    .fa {
+      width: $euiSizeM;
+      height: $euiSizeM;
+      cursor: pointer;
+    }
+  }
+
+  .shard-details-right {
+    color: $euiColorDarkShade;
+    float: right;
+    text-align: right;
+    margin-bottom: $euiSizeXS;
+  }
+
+  .shard {
+    margin-top: $euiSizeS;
+    min-width: 100%;
+    display:inline-block;
+    border: none;
+  }
+
+  .index {
+    margin-top: $euiSize;
+    min-width: 100%;
+    padding: $euiSizeS;
+    display: inline-block;
+  }
+
+  .tv-row:hover {
+    outline: solid 1px $euiColorMediumShade;
```

Comment:
Prolly doesn't need to be so dark. I'd go with `border-bottom: $euiBorderThin;` again